ns1/__init__.py

```diff
@@ -299,27 +299,16 @@ def createAddress(self, prefix, type, network_id, callback=None, errback=None, *
         address = ns1.ipam.Address(self.config, prefix=prefix, type=type, network=network)
         return address.create(callback=callback, errback=errback, **kwargs)
 
-    def loadScopeGroupbyID(self, id, callback=None, errback=None):
+    def loadScopeGroup(self, id, callback=None, errback=None):
         """
-        Load an existing Scope Group by ID into a high level Scope Group object
+        Load an existing Scope Group into a high level Scope Group object
 
         :param int id: id of an existing ScopeGroup
         """
         import ns1.ipam
         scope_group = ns1.ipam.Scopegroup(self.config, id=id)
         return scope_group.load(callback=callback, errback=errback)
 
-    def loadScopeGroupbyName(self, name, service_group_id, callback=None, errback=None):
```

Comment:
Removed, this is no longer consistent since name is not globally unique and service group ID is not returned.